### PR TITLE
Ready: Add org.pymol.PyMOL

### PR DESCRIPTION
**Pull request by kevinsmia1939**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** No respond yet https://github.com/schrodinger/pymol-open-source/issues/172
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
I don't understand the error.

**Comment by tinywrkb** [comment]
This is strange, it looks like the CI doesn't find appstream-util. Try running the bot again.

**Comment by tinywrkb** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
It is ready now. Upstream dev are not interest to maintain 3rd party package.

File: org.pymol.PyMOL.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+ <id>org.pymol.PyMOL.desktop</id>
```
Comment (hfiguiere) [inline org.pymol.PyMOL.metainfo.xml:3]:
```suggestion
 <id>org.pymol.PyMOL</id>
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
> It is ready now. Upstream dev are not interest to maintain 3rd party package.

Did you submit the appdata and desktop file upstream? They are not specific to Flatpak.

File: org.pymol.PyMOL.metainfo.xml (hunk 1 above)
Comment (kevinsmia1939) [reply org.pymol.PyMOL.metainfo.xml]: thanks

**Comment by kevinsmia1939** [comment]
> > It is ready now. Upstream dev are not interest to maintain 3rd party package.
> 
> Did you submit the appdata and desktop file upstream? They are not specific to Flatpak.

Not yet, but I will sent pr to them.

**Comment by kevinsmia1939** [comment]
bot, build org.pymol.PyMOL

**Comment by kevinsmia1939** [comment]
It should be good now.

**Pull request closed without merging**